### DISCUSSION TITLE
How can I override --vds-aspect-ratio-min-height with HTML CDN version

### DISCUSSION THREAD

**Discussion by QuentinGab** [vidstack/player#710] [Q&A] [answered]
Hi,
The default value of --vds-aspect-ratio-min-height is 150px, but I need to override this to a smaller value.

As I use the CDN web component version and the `AspectRatioElement` do not provid a part attribute, it seems to be impossible to override the css --vds-aspect-ratio-min-height variable of the div.container element from the outside (https://github.com/vidstack/player/blob/main/packages/player/src/ui/aspect-ratio/AspectRatioElement.ts#L42).

I would be happy to make a PR to add `part="container"`, or I am missing a other way to achieve this ?

**Comment by mihar-22** [comment, marked as answer]
The `min-height` attribute or `minHeight` property will update the underlying CSS variable for you.

```html
<vds-aspect-ratio min-height="100%" />
```

**Reply by QuentinGab** [reply]
Oh god that was easy, sorry for bothering you thx !!
